pwiz_tools/Skyline/Executables/AssortResources/CsProjFile.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using System.Linq;
 using System.Xml.Linq;
+#nullable enable
 
```

Comment:
@nickshulman Any problem with adding these?

Reply:
Looks good to me.